replapi_it/core.py

```diff
@@ -1,5 +1,4 @@
-from .post import _Post
```

Comment:
Is this what would be considered a "loader" for all the classes?

Reply:
Yes. Right now it doesn't have much functionality, but it will eventually!

Reply:
Sweet :).